win32ss/user/user32/misc/exticon.c

```diff
@@ -421,8 +421,14 @@ static UINT ICO_ExtractIconExW(
 
 	sig = USER32_GetResourceTable(peimage, fsizel, &pData);
 
-/* NE exe/dll */
+/* NE exe/dll: IMAGE_OS2_SIGNATURE
+   .ICO file: 1
+ */
+#ifdef __REACTOS__
+    if (sig==IMAGE_OS2_SIGNATURE || sig==1)
```

Comment:
I took this complete change from exticon.c in current wine-10.0, but it is actually not needed.
I intend to remove this change shortly. It was originally removed in this commit:
https://github.com/reactos/reactos/commit/201e5c8 dated 3-Feb-2016.
It was committed by Jim Tabor with a description that the patch was by Manuel.
It referenced CORE-10789 and was committed in Revision 70685.
I apologize for this misdirection. I should have researched this previously.